client/post-editor/editor-notice/index.jsx

```diff
@@ -84,135 +85,59 @@ export class EditorNotice extends Component {
-								>
-									{ site.title }
-								</a>
-							),
-							a: (
-								<a href={ `/page/${ site.slug }` } onClick={ this.handlePillAddPagePromptClick } />
-							),
-						},
-						comment:
-							'Editor: Message displayed when a page is published, with a link to the site it was published on.',
-					} );
-				}
-
-				if ( 'post' !== type && typeLabel ) {
-					return translate( '%(typeLabel)s published on {{siteLink/}}!', {
-						args: { typeLabel },
+					return translate( 'Page published on {{pageLink/}}! {{a}}Add another page{{/a}}', {
 						components: {
-							siteLink: (
-								<a
-									href={ site.URL }
-									target="_blank"
-									rel="noopener noreferrer"
-									onClick={ this.handlePillExternalClick }
-								>
+							pageLink: (
+								<a href={ postUrl } onClick={ this.handleViewPostClick }>
 									{ site.title }
 								</a>
 							),
+							a: <a href={ `/page/${ site.slug }` } onClick={ this.handleAddPagePromptClick } />,
 						},
-						comment:
-							'Editor: Message displayed when a post of a custom type is published, with a link to the site it was published on.',
+						comment: 'Editor: Message displayed when a page is published, with a link to the page.',
 					} );
 				}
 
-				return translate( 'Post published on {{siteLink/}}!', {
+				return translate( '%(typeLabel)s published on {{postLink/}}!', {
+					args: { typeLabel: typeLabel },
 					components: {
-						siteLink: (
-							<a
-								href={ site.URL }
-								target="_blank"
-								rel="noopener noreferrer"
-								onClick={ this.handlePillExternalClick }
-							>
+						postLink: (
+							<a href={ postUrl } onClick={ this.handleViewPostClick }>
 								{ site.title }
 							</a>
 						),
 					},
 					comment:
-						'Editor: Message displayed when a post is published, with a link to the site it was published on.',
+						'Editor: Message displayed when a post or post of a custom type is published, with a link to the post.',
 				} );
 
 			case 'scheduled':
 				if ( ! site ) {
-					if ( 'page' === type ) {
-						return translate( 'Page scheduled!' );
-					}
-
-					if ( 'post' !== type && typeLabel ) {
-						return translate( '%(typeLabel)s scheduled!', { args: { typeLabel } } );
-					}
-
-					return translate( 'Post scheduled!' );
-				}
-
-				if ( 'page' === type ) {
-					return translate( 'Page scheduled for %(formattedPostDate)s!', {
-						args: { formattedPostDate },
-						comment:
-							'Editor: Message displayed when a page is scheduled, with the scheduled date and time.',
-					} );
-				}
-
-				if ( 'post' !== type && typeLabel ) {
-					return translate( '%(typeLabel)s scheduled for %(formattedPostDate)s!', {
-						args: { typeLabel, formattedPostDate },
-						comment:
-							'Editor: Message displayed when a post of a custom type is scheduled, with the scheduled date and time.',
-					} );
+					return translate( '%(typeLabel)s scheduled!', { args: { typeLabel } } );
```

Comment:
When does this scenario exist? When is `site` false? (Same comment holds for similar cases.)

Reply:
That's a great question - I'm not aware of a scenario, but wonder if it could be some sort of edge-case fallback for Jetpack or maybe the desktop app.

Reply:
Hmm. If we are going to leave it in, would be nice to keep the timestamp in the notice. I'd say to add that back in for now and we can look at totally removing this case in the future once we do some more testing to see when/if this scenario ever actually occurs.

Reply:
Good point… with the switch to `PostLink`, a lot of these messages don't utilize the site prop. I've removed the unnecessary ones.